coalib/output/ConsoleInteraction.py

```diff
@@ -37,6 +39,7 @@
                PrintDebugMessageAction(),
                PrintMoreInfoAction(),
                ShowPatchAction()]
```

Comment:
I think this should be converted to a tuple for immutability, also allows you to get rid of the deepcopy below